Carousel button import is incorrect

Thank you to @dennismphil for telling me about this issue.

```
> webpack --mode production

   699 modules

ERROR in ./node_modules/@salesforce/design-system-react/module/components/carousel/private/auto-play-button.js
Module not found: Error: Can't resolve '../../../../components/button' in '/Users/dphilip/GoogleDrive/workspace/einstein-platform-console/node_modules/@salesforce/design-system-react/module/components/carousel/private'

ERROR in ./node_modules/@salesforce/design-system-react/module/components/carousel/private/previous-next-carousel-navigator.js
Module not found: Error: Can't resolve '../../../../components/button' in '/Users/dphilip/GoogleDrive/workspace/einstein-platform-console/node_modules/@salesforce/design-system-react/module/components/carousel/private'
```

--- a/components/carousel/private/auto-play-button.jsx
+++ b/components/carousel/private/auto-play-button.jsx
@@ -4,7 +4,7 @@
 // ### React
 import React from 'react';
 import PropTypes from 'prop-types';
-import Button from '~/components/button';
+import Button from '../../../components/button';
 import { CAROUSEL_AUTOPLAY_BUTTON } from '../../../utilities/constants';
 
 /**

